lib/color-formatters.js

```diff
@@ -43,3 +45,49 @@ function floorCount(value, yellow, yellowgreen, green) {
   }
 }
 exports.floorCount = floorCount;
+
+function colorScale(steps, colors, reversed) {
+  if (steps === undefined) {
+    throw Error('When invoking colorScale, steps should be provided.');
+  }
+
+  const defaultColors = {
+    1: ['red', 'brightgreen'],
+    2: ['red', 'yellow', 'brightgreen'],
+    3: ['red', 'yellow', 'green', 'brightgreen'],
+    4: ['red', 'yellow', 'yellowgreen', 'green', 'brightgreen'],
+    5: ['red', 'orange', 'yellow', 'yellowgreen', 'green', 'brightgreen'],
+  };
+
+  if (typeof colors === 'undefined') {
+    if (steps.length in defaultColors) {
+      colors = defaultColors[steps.length];
+    } else {
+      throw Error(`No default colors for ${steps.length} steps.`);
```

Comment:
These should be `throw new Error`, not `throw Error`. Invoking constructors without the `new` keyword is deprecated.

Reply:
I can't find a reference supporting this. I used to write `new Error` but stopped because it's extra tokens for seemingly no value. The ES5 spec [specifically permits this](http://es5.github.io/#x15.11.1). Is there an ES6 reference that says otherwise?

I only bring it up because I'd _enforce_ what you're asking for if I had a good reference.

Reply:
Ah, interesting @paulmelnikow! That's still in the ES2015 spec:

> When Error is called as a function rather than as a constructor, it creates and initializes a new Error object. Thus the function call Error(…) is equivalent to the object creation expression new Error(…) with the same arguments.

I guess it's okay then. I was basing my info off an internal lint rule we have at Facebook, but perhaps the info is outdated/incorrect.